design/ceph/object-store-zone.md

```diff
@@ -0,0 +1,183 @@
+
+### Deleting Zone through Toolboxes
+
+The Rook toolbox is a container with common tools used for debugging and testing a Rook Ceph cluster by hand via the CLI.
+
+To launch and configure toolboxes see https://rook.io/docs/rook/master/ceph-toolbox.html
+
+The Rook toolbox can also modify the Ceph Multisite state via the radosgw-admin command.
+
+There are two scenarios possible when deleting a zone.
+The following commands, run via toolboxes, deletes the zone if there is only one zone in the zonegroup.
+
+```bash
+# radosgw-admin zone rm --rgw-zone=zone-z
+# radosgw-admin period update --commit
+```
+
+In the other scenario, there are more than one zones in a zonegroup.
+
+Care must be taken when changing which zone is the master zone.
+
+Please read the following documentation before running the below commands: 
+
+https://docs.ceph.com/docs/master/radosgw/multisite/#changing-the-metadata-master-zone
+
+The following commands, run via toolboxes, remove the zone from the zonegroup first, then delete the zone.
+
+```bash
+# radosgw-admin zonegroup rm --rgw-zone=zone-z
+# radosgw-admin period update --commit
+# radosgw-admin zone rm --rgw-zone=zone-z
+# radosgw-admin period update --commit
+```
+### Changing the Master Zone through Toolboxes
+
+Similar to deleting zones, the Rook toolbox can also change the master zone in a zonegroup.
+
+```bash
+# radosgw-admin zone modify --rgw-zone=zone-z --master
+# radosgw-admin zonegroup modify --rgw-zonegroup=zonegroup-b --master
+# radosgw-admin period update --commit
+```
+
+## Object Store Zone CRD
+
+The object store zone settings are exposed to Rook as a Custom Resource Definition (CRD). The CRD is the Kubernetes-native means by which the Rook operator can watch for new resources.
+
+The name of the resource provided in the `metadata` section becomes the name of the zone.
+
+The following variables can be configured in the object-store-zone resource.
+
+- `zonegroup`: The zonegroup named in the `zonegroup` section of the object-store-realm resource the zone is a part of.
+- `isMaster`: This boolean value signifies whether the zone is master or not. Only one zone can be master. Any object-store-zone resource created as master after a master zone has been created, will not be created.
+
+```yaml
+apiVersion: ceph.rook.io/v1alpha1
+kind: CephObjectStorezone
+metadata:
+  name: zone-b
+  namespace: rook-ceph
+spec:
+  realm: my-realm
+  isMaster: True
+```
+
+### Pools
+
+The pools are the backing data store for the object stores in the zone and are created with specific names to be private to a zone.
+As long as the `zone` config option is specified in the object-store's config, the object-store will use pools defined in the object-store-zone's configuration.
+Pools can be configured with all of the settings that can be specified in the [Pool CRD](/Documentation/ceph-pool-crd.md).
```

Comment:
are pools named resources?
are they specific to the cluster/zone/zonegroup/realm?
if they are named objects, could they be referenced instead of being embedded?